src/runtime-rs/crates/hypervisor/src/device/driver/vfio.rs

```diff
@@ -44,35 +40,6 @@ const INTEL_IOMMU_PREFIX: &str = "dmar";
 const AMD_IOMMU_PREFIX: &str = "ivhd";
 const ARM_IOMMU_PREFIX: &str = "smmu";
 
-lazy_static! {
```

Comment:
Since the macro `update_pcie_device` can achieve the same functionality, the previous related code is no longer needed, so it needs to be deleted in this commit.